internal/broker/config.go

```diff
@@ -86,11 +168,50 @@ func parseConfigFile(cfgPath string) (userConfig, error) {
 		cfg.clientSecret = oidc.Key(clientSecret).String()
 	}
 
-	users := iniCfg.Section(usersSection)
-	if users != nil {
-		cfg.homeBaseDir = users.Key(homeDirKey).String()
-		cfg.allowedSSHSuffixes = strings.Split(users.Key(sshSuffixesKey).String(), ",")
-	}
+	populateUsersConfig(&cfg, iniCfg.Section(usersSection))
 
 	return cfg, nil
 }
+
+func (uc *userConfig) ShouldRegisterOwner() bool {
+	return uc.ownerAllowed && uc.firstUserBecomesOwner && uc.owner == ""
+}
+
+func (uc *userConfig) RegisterOwner(cfgPath, userName string) error {
```

Comment:
This also doesn't need to be exported

```suggestion
func (uc *userConfig) registerOwner(cfgPath, userName string) error {
```